tests/unit/h/views/account_signup_test.py

```diff
@@ -22,19 +21,17 @@ def test_post_returns_errors_when_validation_fails(
             "privacy_accepted": "true",
             "comms_opt_in": "false",
         }
-        mocker.spy(views, "get_csrf_token")
```

Comment:
I replaced this (which was repeated in several tests) with a normal `get_csrf_token` patch fixture below.